### PR TITLE
Use POST verb for shutdown example

### PR DESCRIPTION
**Pull request by wcs1only**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

**Review by AaronCrawfis — APPROVED** [review]

**Review by artursouza — APPROVED** [review]

File: daprdocs/content/en/getting-started/install-dapr-selfhost.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ dapr --version
 
 Output should look like this:
 ```
-CLI version: 1.1.0
-Runtime version: 1.1.0
+CLI version: 1.1.1
```
Comment (mukundansundar) [inline daprdocs/content/en/getting-started/install-dapr-selfhost.md]: this is still 1.1.0

File: daprdocs/content/en/operations/hosting/self-hosted/self-hosted-upgrade.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ description: "Follow these steps to upgrade Dapr in self-hosted mode and ensure
...
 
-1. Ensure you are using the latest version of Dapr (v1.1.0) with:
+1. Ensure you are using the latest version of Dapr (v1.1.1) with:
 
    ```bash
    $ dapr --version
    
-   CLI version: 1.1.0
-   Runtime version: 1.1.0
+   CLI version: 1.1.1
```
Comment (mukundansundar) [inline daprdocs/content/en/operations/hosting/self-hosted/self-hosted-upgrade.md]: still 1.1.0

**Review by mukundansundar — APPROVED** [review]

**Review by mukundansundar — COMMENTED** [review]
Can you add the shutdown API to the API docs? https://github.com/dapr/docs/tree/v1.1/daprdocs/content/en/reference/api

**Review by AaronCrawfis — APPROVED** [review]

**Comment by wcs1only** [comment]
> Can you add the shutdown API to the API docs? https://github.com/dapr/docs/tree/v1.1/daprdocs/content/en/reference/api

Created a separate issue for this: https://github.com/dapr/docs/issues/1386

**Review by mukundansundar — CHANGES_REQUESTED** [review]
update supported versions page

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**